extensions/rbac/verify.go

```diff
@@ -46,6 +49,27 @@ func VerifyGlobalRoleBindingsForUser(t *testing.T, user *management.User, adminC
 	assert.Equal(t, 1, len(grbs.Data))
 }
 
+// VerifyRoleBindingsForUser validates that the corresponding role bindings are created for the user
+func VerifyRoleBindingsForUser(t *testing.T, user *management.User, standardClient *rancher.Client, clusterID, role string) {
+	rblist, err := rbac.ListRoleBindings(standardClient, localcluster, clusterID, metav1.ListOptions{})
```

Comment:
shouldn't the client be admin client? standard user client won't be able to list role bindings from the local cluster.

Reply:
Yes that's correct, I'll change it to admin.